README.md

```diff
@@ -48,16 +48,21 @@ npm run compile
 
 ### Open both repositories in a vscode workspace
 
-```
-code ./vscode-workspaces/multiroot-simple.code-workspace
-```
-
-OR
-
-```
-From the lightning-language-server project, open the file 'multiroot-simple.code-workspace'
-Click button 'Open Workspace'
-```
+```
```

Comment:
Attempt to make this more clear and why there are two ways to open the workspace.